lib/sandbox.js

```diff
@@ -757,6 +757,29 @@ Sandbox.prototype.revokeToken = function (token, cb) {
     return cb ? promise.nodeify(cb) : promise;
 };
 
+Sandbox.prototype.listVersions = function (options, cb) {
+    var request = Superagent
+        .get(this.url + `/api/env/node/modules/${encodeURIComponent(options.name)}`)
```

Comment:
I wonder what the shape of this API would be if and when we support multiple Node.js versions. Something we should do now to future-proof the API?